src/kirby/helpers/color-helper.ts

```diff
@@ -21,6 +21,10 @@ export class ColorHelper {
     return ColorHelper.getColor(name + '-contrast');
   }
 
+  public static getColorBrightness(name: string) {
```

Comment:
```suggestion
  static getColorBrightness(name: string) {
```

Reply:
But then I have to do it on all other methods as well, so I think that would be a candidate for another PR :)